### PR TITLE
Moves zone props to top of entity properties panel

### PR DESCRIPTION
**Pull request by ericrius1**
Previously, zone entity properties were at bottom of entity properties. Now they are at the top
Fixes bug: https://app.asana.com/0/31759584831097/80715240133043

### CODE REVIEW

**Comment by ericrius1** [comment]
test: https://rawgit.com/ericrius1/hifi/zoneEntityProps/examples/edit.js

**Comment by bluetoe** [comment]
I tried it in sandbox and on own domain and never got a populated entity list.

File: examples/html/entityProperties.html [hunk 1]
```diff
@@ -976,6 +976,142 @@
...
+                <div class="input-area">B <input class="coord" type='number' id="property-zone-skybox-color-blue"></div>
+            </div>
+        </div>
+        <div class="zone-section skybox-section property">
+            <div class="label">Skybox URL</div>
+            <div class="value">
+                <input type="text" id="property-zone-skybox-url" class="url">
+            </div>
+        </div>
+    </div>
```
Comment (imgntn) [inline examples/html/entityProperties.html]: @ericrius1 check this

**Comment by hifi-gustavo** [comment]
Build console output for commit: 191e6090e7adaacfbb77eb332adc7bef97a24e79 available at: https://gist.github.com/2cf57bbcbf176a135782<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6966.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6966.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3fff5868e8429d4b09da32244864cf21cb9c0088 available at: https://gist.github.com/2cf57bbcbf176a135782<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6966.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6966.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: c30109ec29130502083ce5f42fc60c88adfc8f1e available at: https://gist.github.com/2cf57bbcbf176a135782<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6966.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6966.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 628158e2fbd0e53b52b3f9a8c5454d426e8dd1a7 available at: https://gist.github.com/2cf57bbcbf176a135782<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6966.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6966.dmg

**Comment by imgntn** [comment]
@ericrius1 also failing build here

**Comment by imgntn** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 628158e2fbd0e53b52b3f9a8c5454d426e8dd1a7 available at: https://gist.github.com/2cf57bbcbf176a135782<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6966.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6966.dmg

**Comment by imgntn** [comment]
lgtm win10

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6be550a144aa4ff8b25f4bf1baa8c3cc5c1d29a4 available at: https://gist.github.com/2cf57bbcbf176a135782<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6966.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6966.dmg

**Pull request merged**